recipe/common.php

```diff
@@ -18,13 +18,32 @@
 set('clear_paths', []);         // Relative path from deploy_path
 set('clear_use_sudo', true);    // Using sudo in clean commands?
 
+
+/**
+ * Composer options
+ */
+env('composer_action', function () {
+    return env('env') === 'prod' ? 'install' : 'update';
```

Comment:
Agree, install!
> 6 июня 2016 г., в 17:10, Oanh Nguyen <notifications@github.com> написал(а):
> 
> In recipe/common.php <https://github.com/deployphp/deployer/pull/688#discussion_r65866588>:
> 
> > @@ -18,13 +18,32 @@
> >  set('clear_paths', []);         // Relative path from deploy_path
> >  set('clear_use_sudo', true);    // Using sudo in clean commands?
> >  
> > +
> > +/**
> > + * Composer options
> > + */
> > +env('composer_action', function () {
> > +    return env('env') === 'prod' ? 'install' : 'update';
> I don't think using composer update is good action.
> It will update your vendors and composer.lock file.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/deployphp/deployer/pull/688/files/0e75d00bdc2acb9126aad9ad104032562aab76f7#r65866588>, or mute the thread <https://github.com/notifications/unsubscribe/AAInsFlgoFwW79Zt-6aKiyKXGEZcPhe9ks5qI_IngaJpZM4IuZkt>.
>